text.go

```diff
@@ -70,8 +71,7 @@ func expandPath(c *gabs.Container, token *list.Element, res *bytes.Buffer) {
 		return
 	}
 	children := c.ChildrenMap()
-	child, ok := children[token.Value.(string)]
-	if ok {
+	if child, ok := children[token.Value.(string)]; ok {
 		p, action, err := getPrefix(child)
```

Comment:
One more change: you'll notice the `getPrefix` returns an action. I propose updating that function to check for the wildcard and return a new `pass` enum (see lines 12-15). With that change in place, you'll be able to instead add a new case to the switch statement like so:

```
case pass:
	res.WriteString("/")
	res.WriteString(token)
```

Reply:
I originally wanted to do something like this but the changes would be a little more involved than tweaking the body of `getPrefix`; there would also need to be a special case where it gets invoked not with `children[token.Value.(string)]` but instead with `children[passKey]`. At that point, it seemed like it'd be more readable to make the special case here explicit by implementing it outside `getPrefix` altogether instead of trying to make that function accommodate two moderately different purposes. WDYT?

Reply:
LGTM, totally fair.